packages/xrpc-server/src/server.ts

```diff
@@ -222,8 +223,12 @@ export class Server {
             output?.encoding === 'json'
           ) {
             res.status(200).json(output.body)
-          } else if (output) {
+          } else if (output?.body instanceof Readable) {
             res.header('Content-Type', output.encoding)
+            res.status(200)
+            forwardStreamErrors(output.body, res)
+            output.body.pipe(res)
```

Comment:
still confused when to use `pipeline` vs this tbh 😅

Reply:
Seems good as-is to me here! 👍